tests/unittests/entityfortest_api_result.py

```diff
@@ -25,8 +25,9 @@ def read(length: int = 20, offset: int = 0,
         if key not in entity_attributes:
             continue
 
-        filters[key] = value.split(',') \
+        filters[key] = value.split(',', 1) \
                        if len(str(value).split(',')) > 1 \
```

Comment:
```suggestion
                       if str(value).count(',') > 1 \
```
Acredito que fique mais performático, o que acha ?

Reply:
Deve mudar pra >=